docs/workflows/faq.md

```diff
@@ -93,3 +93,11 @@ class MyComponent extends Component {
 
 export const defaultView = () => <MyComponent initialState={} />;
 ```
+
+### Working with community addons
```

Comment:
```suggestion
### Which community addons are compatible with the latest version of Storybook?
```

WDYT? Rewrote this as a question because it's a FAQ :)

Reply:
@shilman works for me. Reads better :) ! Going to accept the proposed change